docsbuild/content/migrations/identityprovider.md

```diff
@@ -59,6 +59,52 @@ changes:
       trustEmail: true
       storeToken: true
 ```
+## UpdateKeycloakIdentityProvider
+Updates an existing keycloak identity provider.
```

Comment:
I would add, and I am quite sure this is the case: `Updates an existing keycloak identity provider leaves all not provided fields untouched.`

Reply:
done